packages/cli/src/util/link/git-connect-prompts.ts

```diff
@@ -0,0 +1,83 @@
+import { Dictionary } from '@vercel/client';
+import chalk from 'chalk';
+import { Project } from '../../types';
+import Client from '../client';
+import { formatProvider } from '../git/connect-git-provider';
+import list from '../input/list';
+import { Output } from '../output';
+
+export async function promptGitConnectSingleUrl(
+  client: Client,
+  output: Output,
+  project: Project,
+  remoteUrl: string,
+  replace = false
+) {
+  if (replace) {
+    const currentRepoPath = `${project.link!.org}/${project.link!.repo}`;
+    const currentProvider = project.link!.type;
+    output.log(
+      `Found Git remote url ${chalk.cyan(
+        remoteUrl
+      )}, which is different from the connected ${formatProvider(
+        currentProvider
+      )} repository ${chalk.cyan(currentRepoPath)}.`
+    );
+  } else {
+    output.log(`Found local Git remote URL: ${chalk.cyan(remoteUrl)}`);
+  }
+  return await list(client, {
```

Comment:
Could be just me, but after accepting the project settings, the command does work and then a few seconds later it prints out 8 lines for the Git remote URL prompt.

For me, it seems like a bit too much to process: the bold text, the `>`, the `●`, the `○`, the colors. What do you think of inserting a line break before prompting?

<img width="488" alt="image" src="https://user-images.githubusercontent.com/97262/183529341-5de3e994-0f72-492f-a9c5-1ca118bd152f.png">

Reply:
I had the same experience. I think a line break somewhere here would be useful. If it goes before the line "Found local Git remote", it keeps that line in the context of the prompt. It may be clearer to have the origin name in these lines.

It could look like:

```
Auto-detected Project Settings (Hexo):
- Build Command: hexo generate
- Development Command: hexo server --port $PORT
- Install Command: `yarn install`, `pnpm install`, or `npm install`
- Output Directory: public
? Want to modify these settings? [y/N] n

> Found local Git remote "origin": https://github.com/EndangeredMassa/boa-v3-hexo.git
> Do you want to connect "origin" to your Vercel project? yes
> Connecting...
> Connected GitHub repository EndangeredMassa/boa-v3-hexo!
✅  Linked to endangeredmassa/test-link-2 (created .vercel)
```

Reply:
I love the suggestion. I think I just addressed this.